src/common/constants.ts

```diff
@@ -19,8 +19,8 @@ export const DEFAULT_MAINNET_SERVER = process.env.NEXT_PUBLIC_MAINNET_API_SERVER
 
 export const MAINNET_ENABLED = process.env.NEXT_PUBLIC_MAINNET_ENABLED === 'true';
 
-export const NETWORK_LIST_COOKIE = 'network-list';
-export const NETWORK_CURRENT_INDEX_COOKIE = 'network-current';
+export const NETWORK_LIST_COOKIE = 'STAGING__NETWORK_LIST';
+export const NETWORK_CURRENT_INDEX_COOKIE = 'STAGING__NETWORK_CURRENT_INDEX';
```

Comment:
Why this change?

Reply:
I will change it before release, but there was some legacy saved data that was interfering with some peoples experience